### PR TITLE
fixed mark as paid error

### PR DESCRIPTION
**Pull request by elraphty**
## Describe your changes

This PR fixes the mark bounty as paid bug.

## Issue ticket number and link
closes [647](https://github.com/stakwork/sphinx-tribes/issues/647)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device

### CODE REVIEW

File: frontend/app/src/store/main.ts [hunk 1]
```diff
@@ -1670,6 +1671,25 @@ export class MainStore {
...
+        mode: 'cors',
+        headers: {
+          'x-jwt': info.jwt,
+          'Content-Type': 'application/json'
+        }
+      });
+
+      return r;
+    } catch (e) {
+      return false;
```
Comment (kevkevinpal) [inline frontend/app/src/store/main.ts:1690]: lets log out the error here since this is a failure

File: handlers/bounty.go [hunk 2]
```diff
@@ -148,9 +149,24 @@ func DeleteBounty(w http.ResponseWriter, r *http.Request) {
 	}
 
 	b, _ := db.DB.DeleteBounty(pubkey, created)
+
+	w.WriteHeader(http.StatusOK)
 	json.NewEncoder(w).Encode(b)
 }
 
+func UpdatePaymentStatus(w http.ResponseWriter, r *http.Request) {
+	createdParam := chi.URLParam(r, "created")
```
Comment (kevkevinpal) [inline handlers/bounty.go:158]: shouldnt we also check the pubkey of the bounty and the person sending the request?

Comment (elraphty) [reply handlers/bounty.go:158]: We removed the auth functionality for now, Once the auth is working perfectly I will add this to it, I will add a todo for now.

File: frontend/app/src/store/main.ts (hunk 1 above)
Comment (elraphty) [reply frontend/app/src/store/main.ts:1690]: Okay

File: handlers/bounty.go (hunk 2 above)
Comment (kevkevinpal) [reply handlers/bounty.go:158]: gotcha sounds good!

**Pull request merged**